src/sklearn_raster/datasets/_base.py

```diff
@@ -157,6 +192,38 @@ def load_swo_ecoplot(
     using all available Landsat imagery. Remote Sensing of Environment. 122:75–91.
     """
     X, y = sknnr.datasets.load_swo_ecoplot(return_X_y=True, as_frame=True)
+    variables = [
+        _VariableMeta("ANNPRE", "Annual precipitation", "ln mm"),
+        _VariableMeta("ANNTMP", "Mean annual temperature", "C"),
+        _VariableMeta("AUGMAXT", "Mean August maximum temperature", "C"),
+        _VariableMeta(
+            "CONTPRE", "Percentage of annual precipitation falling in June-August", "%"
+        ),
+        _VariableMeta(
+            "CVPRE",
+            "Coefficient of variation of mean monthly precipitation of December and "
+            "July",
+        ),
+        _VariableMeta("DECMINT", "Mean December minimum temperature", "C"),
+        _VariableMeta(
+            "DIFTMP",
+            "Difference between mean August maximum and December minimum temperatures",
+            "C",
+        ),
+        _VariableMeta("SMRTMP", "Mean temperature from May-September", "C"),
+        _VariableMeta("SMRTP", "Growing season moisture stress", "C / ln mm"),
+        _VariableMeta("ASPTR", "Cosine transformation of aspect"),
+        _VariableMeta("DEM", "Elevation", "m"),
+        _VariableMeta("PRR", "Potential relative radiation", "%"),
```

Comment:
This is a weird one.  It's basically a bunch of `HILLSHADE` rasters added up together (0-254) for hourly solar azimuths and altitudes for each monthly midpoint.  So it probably does represent some kind of a percent of maximum solar radiation, but we've always called in unitless.  Probably not a big deal.